adjust_proxy_config.py

```diff
@@ -0,0 +1,85 @@
+import sys
+from argparse import ArgumentParser
+from typing import List
+
+import toml
+
+
+def main(cli_args: List[str]):
+    parser = ArgumentParser()
+    parser.add_argument("--file", required=True)
+    parser.add_argument("--network")
+
+    parsed_args = parser.parse_args(cli_args)
+    file = parsed_args.file
+    network = parsed_args.network
+    data = toml.load(file)
+
+    if network == "testnet":
```

Comment:
refactored L18-L76 in a new function with early exits and removed `else` / `elif` branches?

Reply:
Fixed (refactored).